CONTRIBUTING.markdown

```diff
@@ -1,106 +1,124 @@
-Contribute
-==========
-
-So you've got an awesome idea to throw into Jekyll. Great! Please keep the
-following in mind:
-
-* **Use https://talk.jekyllrb.com for non-technical or indirect Jekyll questions that are not bugs.**
-* **Contributions will not be accepted without tests or necessary documentation updates.**
-* If you're creating a small fix or patch to an existing feature, just a simple
-  test will do. Please stay in the confines of the current test suite and use
-  [Shoulda](https://github.com/thoughtbot/shoulda/tree/master) and
-  [RSpec-Mocks](https://github.com/rspec/rspec-mocks).
-* If it's a brand new feature, make sure to create a new
-  [Cucumber](https://github.com/cucumber/cucumber/) feature and reuse steps
-  where appropriate. Also, whipping up some documentation in your fork's `site`
-  would be appreciated, and once merged it will be transferred over to the main
-  `site`, jekyllrb.com.
-* If your contribution changes any Jekyll behavior, make sure to update the
-  documentation. It lives in `site/_docs`. If the docs are missing information,
-  please feel free to add it in. Great docs make a great project!
-* Please follow the [GitHub Ruby Styleguide](https://github.com/styleguide/ruby)
-  when modifying Ruby code.
-* Please do your best to submit **small pull requests**. The easier the proposed
-  change is to review, the more likely it will be merged.
-* When submitting a pull request, please make judicious use of the pull request
-  body. A description of what changes were made, the motivations behind the
-  changes and [any tasks completed or left to complete](http://git.io/gfm-tasks)
-  will also speed up review time.
-
-Test Dependencies
------------------
-
-To run the test suite and build the gem you'll need to install Jekyll's
-dependencies. Simply run this command to get all setup:
+---
+layout: docs
+title: Contributing
+permalink: /docs/contributing/
+---
```

Comment:
Let's automate this [the way we automate adding `/docs/history/`](https://github.com/jekyll/jekyll/blob/master/rake/site.rake#L108-L112).